Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { TextInput} from 'react-native';
+
+export default class GrowingTextbox extends Component {
```

Comment:
woah, a class. this is uncharted territory. Good shit though

Reply:
Don't worry its copy pasted from stack overflow